MaxText/configs/base.yml

```diff
@@ -247,3 +247,28 @@ decode_sampling_temperature: 1.
 
 eval_interval: -1  # the specific number of train step between eval_step
 target_eval_loss: 0.  # early stop once reaching target eval_loss
+
+# Vertex AI Tensorboard Configurations
+# Set to True to create Vertex Tensorboard instance in GCP
+# Set to False if running via XPK as Vertex Tensorboard and Experiment will be created by XPK
+create_vertex_tensorboard: False
+# Project to create Vertex Tensorboard in if create_vertex_tensorboard=True
+# Project where Vertex Tensorboard to upload data exists if upload_data_to_tensorboard=True
+# Set to blank if running via XPK as XPK will set this as an environment variable
+vertex_tensorboard_project: ""
```

Comment:
Shouldn't this typically not be necessary -- it will inherit from the current project?

Reply:
(If not local?)

Reply:
Yes, added this to cater local development as well.

Reply:
But isn't local development in a project? Just trying to understand.

Reply:
feature sounds great but now the comment is wrong right?

Reply:
Updated the comment.